float32 arm_pos_x
float32 arm_pos_y
float32 arm_pos_z
float32 arm_q1
float32 arm_q2
float32 arm_q3
float32 arm_q4
float32 gripper

